models/ChatbotMessage.ts

```diff
@@ -0,0 +1,10 @@
+const ChatbotMessageSchema = new Schema({
+  body: { type: String, required: true },
+  images: { type: Array, required: false },
+  nextMessages: { type: Array, required: true },
```

Comment:
https://mongoosejs.com/docs/schematypes.html#maps

Great work on the rest of this PR, everything looks really good. One small adjustment here: I talked to Alex about this issue during our 1-1, but I was thinking about we should go about actually splitting into the correct message. An array is one way to do it, but we'd somehow need to transmit and store a correct index, which is possible in some way, but seems very clunky as I think about it more.

I think the best way to do it is through a map (or object, but it's referred to as a map in Mongoose documentation oops) which I've linked above. Additionally, be sure to type the content of arrays as done on line 54 in this previous project here: https://github.com/GTBitsOfGood/gen-soln/blob/develop/server/models/nonprofit.js using `type: Schema.ObjectId, ref: 'ChatbotMessage'`.